gen/definitions/hosts.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: Hosts
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/hosts
+res_description: >-
+  This plural resource manages a bulk of Hosts.
+  The FMC API supports quick bulk creation of this resource. Deletion of this resource is done one-by-one or in bulk, depending of FMC version. Modification is always done one-by-one.
+  Updating/deleting `fmc_hosts` can thus take much more time than creating it
+doc_category: Objects
+attributes:
+  - model_name: items
+    type: Map
+    description: >-
+      Map of hosts. The key of the map is the name of the individual Host. Renaming Hosts in bulk
+      is not yet implemented.
+    map_key_example: hosts_1
+    mandatory: true
+    attributes:
+      - model_name: id
+        type: String
+        resource_id: true
+        description: UUID of the managed Host.
+        exclude_example: true
+        exclude_test: true
+      - model_name: description
+        type: String
+        description: Optional user-created description.
+        example: "My Host 1"
+      - model_name: overridable
+        type: Bool
+        description: Indicates whether object values can be overridden.
+        exclude_example: true
+        test_value: "true"
+      - model_name: value
+        tf_name: ip
+        type: String
+        mandatory: true
+        description: IP of the host.
+        example: 10.1.1.1
+      - model_name: type
+        type: String
+        description: Type of the object; this value is always 'Host'.
+        default_value: Host
```

Comment:
I think this probably carried over from another resource, but if this should not be configurable, is there a reason to not make this a hardcoded value?

```yaml
      - model_name: type
        type: String
        value: Host
```

Reply:
From my observation, once the value is hardcoded, it no longer becomes an attribute of the object. Having that attribute may become handy, as some references require ID + type. I'm not saying this is the case in this particular case, but to keep it consistent across object types perhaps we could keep it.

Reply:
I left it as it is for now. Will try to make it read-only at next stage.